src/test/java/macrobase/analysis/outlier/MovingAverageTest.java

```diff
@@ -4,15 +4,19 @@
 
 import java.util.ArrayList;
 
+import macrobase.conf.MacroBaseConf;
 import macrobase.datamodel.TimeDatum;
 
 import org.apache.commons.math3.linear.ArrayRealVector;
 import org.junit.Test;
 
+import com.google.common.collect.ImmutableMap;
+
 public class MovingAverageTest {
     @Test
     public void simpleTest() {
-        MovingAverage ma = new MovingAverage(3);
+        MacroBaseConf conf = new MacroBaseConf(ImmutableMap.of(MacroBaseConf.TUPLE_WINDOW, 3));
```

Comment:
Slightly prefer a `new MacroBaseConf().set(MacroBaseConf.TUPLE_WINDOW, 3)`. May require making `MacroBase.set` return `this`.